synapse/lib/storm.py

```diff
@@ -1319,3 +1320,12 @@ def _stormOperFromTags(self, query, oper):
                 [query.add(n) for n in nodes]
 
         return
+
+    def _stormOperGuid(self, query, oper):
+        args = oper[1].get('args')
+        core = self.getStormCore()
+
+        for iden in args:
+            node = core.getTufoByIden(iden)
```

Comment:
This is changed - we'll need to consider this while doing storage layer separation though since currently on psql optimizes that lift.